builds/e2e/longhaul.yaml

```diff
@@ -349,4 +349,128 @@ jobs:
           snitch.storage.account: '$(snitch.storage.account)'
           snitch.storage.masterKey: '$(StorageAccountMasterKeyStress)'
           loadGen.message.frequency: '$(loadGen.message.frequency.amd64)'
-         
\ No newline at end of file
+          
+################################################################################
+  - job: setup_iotuap_amd64_moby
+################################################################################
+    displayName: Windows IoT Core AMD64 Moby (Setup Only)
+    condition: eq(variables['run.iotuap.amd64.moby'], 'true')
+    pool:
+      name: $(pool.name)
+      demands:
+        - longhaul_iotuap_x64 -equals true
```

Comment:
Please work with Chanisha; to follow the convention she created; and try to reuse template she is going to create.  You can refer to this PR, https://github.com/Azure/iotedge/pull/1296.

Reply:
I rebased my branch to include Chanisha's changes after merged to Azure/iotedge.